inductiva/fluids/wind_tunnel/README.md

```diff
@@ -56,7 +56,7 @@ The simulation parameters available for the user to configure are:
 - `num_iterations`: Set the maximum number of iterations for the iterative algorithm to converge.
 - `resolution`: Controls the resolution of the meshing that is done prior to the simulation. The higher the resolution, the finer the meshing. Available options: "high", "medium", "low".
 
-Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
+Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `n_cores`.
```

Comment:
is n_cores still there?

Reply:
Yes, for OpenFOAM we need extra care due to changing the commands based on the n_cores. This is an extra step that needs a more careful consideration to move to the backend, since it interferes with the low-level simulation as well.